src/templates/static/about/careers.jsx

```diff
@@ -44,12 +44,26 @@ const ColumnSM = ({ gr, header, paragraph }) => (
     </div>
 );
 
+const ColumnValues = ({ gr, icon, header }) => (
+    <div className={`gr-${gr ? `${gr} gr-12-m` : '3 gr-6-p gr-12-m'}`}>
+        <div className='values-box'>
+            <img className='responsive values-box-icon' src={it.url_for(`images/pages/careers/${icon}.svg`)} />
+            <div className='fill-text'>
+                <h4 className='values-box-subheader'>{header}</h4>
+            </div>
+        </div>
+    </div>
+);
 
-const ColumnLG = ({ center, image, text }) => (
-    <div className={`gr-6 gr-12-m${center ? ' center-text' : ''}`}>
+
+const ColumnLG = ({ center, image, text, link }) => (
+    <div className={`gr-4 gr-12-m${center ? ' center-text' : ''}`}>
+        <div>
+            <p className='column-margin'>{text}</p>
+        </div>
         <img className='responsive' src={it.url_for(`images/pages/careers/${image}.jpg`)} />
-        <div className='white-bg-color'>
-            <p className='column-margin fill-text'>{text}</p>
+        <div className='fill-text'>
+            <a href={link}><span>{it.L('Learn more')}</span></a>
```

Comment:
perhaps can add a check if link exists then return `<a/>` else return `<p/>`

Reply:
@kellybinary , is this better? Since if we don't have `link` prop it's pointless to return anything else in place.

```{!!link &&
            <div className='fill-text'>
                <a href={link}><span>{it.L('Learn more')}</span></a>
            </div>
        }```

Reply:
ahh right, but what about the `Learn more` text? shouldn't it be `{it.L(text)}` instead?

Reply:
`text` prop is for the header on top of the image. The `button link` will always use `Learn More` in this use case.